src/main/resources/static/js/common.js

```diff
@@ -64,12 +64,12 @@ AjaxHttpSender.prototype.sendAjax = function (contextPath, type, data, callback,
 }
 // /Stop ## Global AJAX Sender function ##################################
 
-function formatUrl(host, port, https, contextPath) {
-    var http = "http";
-    var useHttps = (https == true || https == "https");
+function formatUrl(host, port, useHttps, contextPath) {
+    var scheme = "http";
```

Comment:
I think protocol is the used word to describe HTTP/HTTPS

Reply:
https://en.wikipedia.org/wiki/Uniform_Resource_Identifier

Reply:
The scheme includes [http://] while the protocol is [http]://. In common tongue you talk about which protocol you want to use when doing a request so i still think it's confusing to use scheme.